Makefile

```diff
@@ -194,22 +204,26 @@ ux-tarball: build-ux dist ## Build website asset tarball
 		--directory=src/js/engagement_view/build \
 		.
 
-.PHONY: lambdas
-lambdas: lambdas-rust lambdas-js lambdas-python ## Generate all lambda zip files
-
-.PHONY: lambdas-rust
-lambdas-rust: ## Build Rust lambda zips
-	$(DOCKER_BUILDX_BAKE) -f docker-compose.lambda-zips.rust.yml
-	docker-compose -f docker-compose.lambda-zips.rust.yml up
-
-.PHONY: lambdas-js
-lambdas-js: ## Build JS lambda zips
-	$(DOCKER_BUILDX_BAKE) -f docker-compose.lambda-zips.js.yml
-	docker-compose -f docker-compose.lambda-zips.js.yml up
-
-.PHONY: lambdas-python
```

Comment:
The merge pipeline uses these targets and files; you'll have to update it accordingly.